app/root.tsx

```diff
@@ -3,20 +3,59 @@ import type {MetaFunction, LinksFunction, LoaderFunction} from '@remix-run/cloud
 import styles from '~/root.css'
 
 import {useLoaderData} from '@remix-run/react'
+import {getStateEntries} from '~/hooks/stateModifiers'
+import {loadQuestionDetail} from '~/server-utils/stampy'
 
-const title = 'Stampy'
-const description = 'AI Safety FAQ'
+/*
+ * Transform the given text into a meta header format.
+ *
+ * In practice, this means stripping out any HTML tags and limiting its length
+ */
+const metafyText = (text: string | null, defaultText: string) => {
+  if (!text || text.length == 0) return defaultText
+
+  let cleaned = text
+  // Totally remove any P, A or IFRAME tags, but leave the contents. Any other
+  // tags will be ignored for now
+  const removals = [/<\/?p.*?>/gi, /<\/?a.*?>/gi, /<\/?iframe.*?>/gi]
```

Comment:
I guess the regex is good for now, but when @plexish will test it more and asks for changes to the removals (or other modifications), it will be even better to add unit tests for the input/output examples to this function

IMHO this is the first big feature that the (_implicit manual regression testing during_) local development is unlikely to discover regression bugs